proto/relay.proto

```diff
@@ -33,10 +16,6 @@ service AppService {
   rpc AppConnect(stream RequestStream) returns (stream ResponseStream);
 }
 
```

Comment:
Do we still need `OrbService` and `AppService`?

Reply:
Probably not. I was just scared to remove them because I'm not sure what's the current status on the server with respect to services. Should I nuke them too?

Reply:
I think we don't

Reply:
nuke them

Reply:
<img src="https://media4.giphy.com/media/3o7abwbzKeaRksvVaE/giphy.gif"/>